src/examples/local-state-key-change.js

```diff
@@ -0,0 +1,74 @@
+// useEffect: persistent state
+// 💯 flexible localStorage hook - changing the key in localStorage
+// http://localhost:3000/isolated/examples/local-state-key-change.js
+
+import * as React from 'react'
+
+function useLocalStorageState(
```

Comment:
Instead of hard-coding this, could we import it from [here](https://github.com/kentcdodds/react-hooks/blob/861594b4d999368c037604ad32e222f14d15465d/src/utils.js#L10)?

Reply:
Yes! I completely missed this utils function as I have not yet gotten there. Great suggestion and thank you!